rdr_service/genomic/genomic_job_controller.py

```diff
@@ -240,6 +246,148 @@ def ingest_specific_manifest(self, filename):
         except RuntimeError:
             self.job_result = GenomicSubProcessResult.ERROR
 
+    def ingest_member_ids_from_aw1_raw_table(self, member_ids):
+        """
+        Pulls AW1 data from genomic_aw1_raw and loads to genomic_set_member
+        :param member_ids: list of genomic_set_member_ids to ingest
+        :return: ingestion results as string
+        """
+        member_dao = GenomicSetMemberDao()
+        raw_dao = GenomicAW1RawDao()
+
+        # Get member records
+        members = member_dao.get_members_from_member_ids(member_ids)
+
+        update_recs = []
+        completed = []
+        multiples = []
+        missing = []
+
+        for m in members:
+            # add prefix to biobank_id
+            try:
+                pre = self.server_config[config.BIOBANK_ID_PREFIX][0]
+            except KeyError:
+                # Set default for unit tests
+                pre = "A"
+
+            bid = f"{pre}{m.biobankId}"
+
+            # Get Raw AW1 Records for biobank IDs and genome_type
+            try:
+                raw_rec = raw_dao.get_raw_record_from_bid_genome_type(bid, m.genomeType)
+
+            except MultipleResultsFound:
+                multiples.append(m.id)
+
+            except NoResultFound:
+                missing.append(m.id)
```

Comment:
These are great! Need to def use these instead of testing for lengths moving forward. 
Probably easier to do on one line:
`except (MultipleResultsFound, NoResultFound)`